src/test/java/com/microsoft/sqlserver/jdbc/resultset/DataClassificationTest.java

```diff
@@ -0,0 +1,134 @@
+package com.microsoft.sqlserver.jdbc.resultset;
+
+import java.sql.Connection;
```

Comment:
Apply the formatter please

Reply:
Already applied - I don't see any change on doing again